UnitTests/CommonTestUtils/MEF/ExportProviderCache.SimpleAssemblyLoader.cs

```diff
@@ -22,7 +22,9 @@ public Assembly LoadAssembly(string assemblyFullName, string codeBasePath)
                 AssemblyName assemblyName = new(assemblyFullName);
                 if (!string.IsNullOrEmpty(codeBasePath))
                 {
+#pragma warning disable SYSLIB0044 // 'AssemblyName.CodeBase' is obsolete: 'AssemblyName.CodeBase and AssemblyName.EscapedCodeBase are obsolete. Using them for loading an assembly is not supported.'
```

Comment:
What is the final solution here?
Is the suppression good enough for a release?

Reply:
Since this is the test code I didn't bother. But it may be worth investigating at some point.